### DISCUSSION TITLE
Add option to copy JS import code

### DISCUSSION THREAD

**Discussion by dospolov** [tailwindlabs/heroicons#615] [Ideas]
I suggest to add here:
![image](https://user-images.githubusercontent.com/670682/114115151-f9769700-98ea-11eb-8012-5059bc90e598.png)

another option "Copy import". Clicking it will copy next line to buffer:
`import { EyeOffIcon} from '@heroicons/react/outline`

**Comment by RobinMalfait** [comment]
Hey! Doing a big GitHub clean-up this week so going to move this over to a discussion. Still excited to play with this idea though 🥳
